src/components/UI/NumberPicker.js

```diff
@@ -222,7 +223,7 @@ class NumberPicker extends React.Component {
   render() {
     return (
       <Wrapper
-        className={`${this.props.theme ? this.props.theme : ''} ${
+        className={`${this.props.className ?? ''} ${this.props.theme ?? ''} ${
```

Comment:
Not absolutely necessary for this PR, but maybe we should create a helper function for classnames at some point.

I'm thinking of something like
```typescript
function classNames(...classes: Array<string | undefined>): string {
...join into one string, ignoring undefined values
}
```

Reply:
Maybe use this: https://github.com/lukeed/clsx

Reply:
C'mon man, we can do that in 3 lines of code, no need for a library 😬